packages/extension/src/newtab/CustomLinks.tsx

```diff
@@ -54,7 +57,7 @@ export function CustomLinks({
       <SimpleTooltip placement="left" content="Edit shortcuts">
         <Button
           variant={ButtonVariant.Tertiary}
-          icon={<MenuIcon />}
+          icon={<MenuIcon className={shouldUseFeedLayoutV1 && 'rotate-90'} />}
```

Comment:
I noticed the icon is not rotated as it should be based on Figma. Same with the border color. I decided to tackle it since I was already touching the same file. I know it goes against doing the bare minimum, but this was a real OCD cringe for me 👁️

Reply:
No that's great thank u!! Just wasn't sure what the change was about :D Didn't notice we had missed that